src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.hpp

```diff
@@ -24,6 +24,7 @@ class LLMCompiledModel : public ov::npuw::ICompiledModel {
         uint32_t total_size = 0u;
         uint32_t num_stored_tokens = 0u;
         uint32_t dim = 0u;
+        uint32_t max_generation_token_len = 0u;
```

Comment:
Please add it to serialization and bump serialization version

Reply:
Fixed, thanks!